barretenberg/cpp/src/barretenberg/dsl/acir_format/acir_integration.test.cpp

```diff
@@ -543,4 +543,26 @@ TEST_F(AcirIntegrationTest, DISABLED_UpdateAcirCircuit)
     EXPECT_TRUE(prove_and_verify_honk<Flavor>(circuit));
 }
 
+/**
+ * @brief Test recursive honk recursive verification
+ *
+ */
+TEST_F(AcirIntegrationTest, DISABLED_HonkRecursion)
```

Comment:
This is just a convenient way to debug honk recursion without getting `bb` involved